assets/data/aspects/drauvenSkillDeck.json

```diff
@@ -256,5 +256,35 @@
 	"modId": "wildermyth-drauven-pcs",
 	"id": "drauven_pc_preparedShotAlsoPoisons",
 	"effects": [ "drauven_pc_preparedShotAlsoPoisons", "removeAtStartOfTurn"]
-}
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_hunter_drauven_guerilla",
+	"effects": [ "drauven_pc_drauven_guerilla" ],
+	"boostType": "ABILITY_CLASS",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": { "abilityDeckUsage": "hunter", "abilityDeckIcon": "thornfang" }
+},
+{
+	"id": "drauvenDeck_hunter_drauven_guerilla_upgrade",
+	"effects": [ "drauven_pc_drauven_guerilla_upgrade_hit" ],
+	"legacyAbilityChoice": true,
+	"importance": 5,
+	"showInTooltip": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_hunter_drauven_guerilla",
+		"abilityDeckIcon": "thornfang"
+	}
+},
+{
+	"id": "drauvenDeck_hunter_drauven_guerilla_freemove",
```

Comment:
Do we need this as a separate skill choice? If this isn't going to be directly offered as a skill choice, let's move it to another file so that this one only contains things that can be explicitly offered on level-up. It looks like the file `drauvenSkillUpgrades` is already being used for stuff like that :)